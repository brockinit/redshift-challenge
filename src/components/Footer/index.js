import githubSVG from '../../assets/github.svg';
import linkedinSVG from '../../assets/linkedin.svg';

const footerItems = [
  {
    label: 'LinkedIn',
    icon: linkedinSVG,
    link: 'https://www.linkedin.com/in/brocklanoza',
  },
  {
    label: 'Github',
    icon: githubSVG,
    link: 'https://github.com/brockinit',
  },
];

const Footer = () => {
  return (
    <div className="footer-container">
      <div className="footer-container-inner">
        {footerItems.map(item => (
          <div className="footer-item">
            <a href={item.link}>
              <img src={item.icon} alt={item.label} />
            </a>
          </div>
        ))}
      </div>
    </div>
  );
};

export default Footer;
