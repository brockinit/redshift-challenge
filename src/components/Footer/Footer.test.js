import {
  renderToSnapshot,
  isClassVNode,
  isFunctionalVNode,
} from 'inferno-test-utils';
import Footer from './';

test('Footer should be a functional component', () => {
  const isClass = isClassVNode(<Footer />);
  const isFunctional = isFunctionalVNode(<Footer />);
  expect(isClass).toBe(false);
  expect(isFunctional).toBe(true);
});

test('Footer should render correctly', () => {
  const shallowRender = renderToSnapshot(<Footer />);
  expect(shallowRender).toMatchSnapshot();
});
