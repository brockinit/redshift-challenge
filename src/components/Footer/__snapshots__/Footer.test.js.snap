// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer should render correctly 1`] = `
<div
  className="footer-container"
>
  <div
    className="footer-container-inner"
  >
    <div
      className="footer-item"
    >
      <a
        href="https://www.linkedin.com/in/brocklanoza"
      >
        <img
          alt="LinkedIn"
          src="linkedin.svg"
        />
      </a>
    </div>
    <div
      className="footer-item"
    >
      <a
        href="https://github.com/brockinit"
      >
        <img
          alt="Github"
          src="github.svg"
        />
      </a>
    </div>
  </div>
</div>
`;
